lib/Clownface.js

```diff
@@ -389,12 +389,14 @@ class Clownface {
    * Deletes all quads where the current graph pointer contexts are the objects
    *
    * @param {Term|Term[]|Clownface|Clownface[]} predicates
+   * @param {Term|Term[]|Clownface|Clownface[]} subjects
```

Comment:
This is optional, right?

```suggestion
   * @param {Term|Term[]|Clownface|Clownface[]} [subjects]
```

Reply:
You are right. The first argument is also optional, which was missing. Also fixed that.